service-base/src/main/java/org/eclipse/hono/service/HealthCheckServer.java

```diff
@@ -93,14 +92,10 @@ private void prepareHealthCheck() {
      * 
      * @param serviceInstance instance of the service which's checks should be registered.
      */
-    public void registerHealthCheckResources(HealthCheckProvider serviceInstance) {
+    public final void registerHealthCheckResources(HealthCheckProvider serviceInstance) {
```

Comment:
param should be final

since all methods are final now, is this class intended to be extended anyway? if not, it would probably make sense to make the class itself final, instead of the methods. WDYT?